index.html

```diff
@@ -36,6 +36,71 @@
       </nav>
     </div>
   </header>
+  <div class = "section1"> 
+    <div class = "trend"><img src = "assets/rediumpics/trending.png" class="trendingLogo" alt = "trending logo"> <span class = "trending">TRENDING ON REDIUM</span></div>
```

Comment:
Great job adding `alt` for the images ⭐ ⭐

Reply:
Thanks!